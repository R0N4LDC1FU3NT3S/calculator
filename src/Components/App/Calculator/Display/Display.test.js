import React from "react"
import { shallow } from "enzyme"
import Display from "./Display"

describe("Display", () => {
	let wrapper

	beforeEach(() => (wrapper = shallow(<Display displayValue={""} />)))

	it("#0 - Snapshot", () => {
		expect(wrapper).toMatchSnapshot()
	})

	it("#1 - Render correctly", () => {
		expect(wrapper.find("div").length).toEqual(1)
	})

	it("#2 - Render displayValue", () => {
		wrapper.setProps({ displayValue: "test" })
		expect(wrapper.text()).toEqual("test")
	})
})
