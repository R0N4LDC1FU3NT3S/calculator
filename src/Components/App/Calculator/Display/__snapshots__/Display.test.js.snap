// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display #0. Snapshot 1`] = `ShallowWrapper {}`;
